data/json/recipes/ammo/shot.json

```
@@ -263,28 +263,29 @@
     "skill_used": "fabrication",
     "difficulty": 2,
     "skills_required": [ "gun", 1 ],
-    "time": "2 m",
+    "time": "1 m",
     "batch_time_factors": [ 50, 5 ],
     "autolearn": true,
-    "charges": 2,
+    "charges": 1,
     "reversible": true,
-    "using": [ [ "ammo_shot", 2 ] ],
+    "using": [ [ "ammo_shot", 1 ] ],
     "tools": [ [ [ "press", -1 ] ] ],
     "//": "1166 mg gunpowder rounded to 12 100 mg 'pieces'; recipe makes two shells",
+	  "//1": "currently halved inputs (rounding up) and result pending fix of casing duplication",
```

Use space characters for indentation:
```suggestion
    "//1": "currently halved inputs (rounding up) and result pending fix of casing duplication",
```